package.json

```diff
@@ -12,11 +12,12 @@
   "private": true,
   "scripts": {
     "lint": "eslint --quiet \"**/*.js\"",
+    "stylelint": "stylelint --quiet --allow-empty-input \"./packages/stylelint-plugin-ckeditor5-rules/style.css\" --fix",
     "precommit": "lint-staged",
     "changelog": "node ./scripts/changelog.js",
     "release:bump-version": "node ./scripts/bump-version.js",
     "release:publish": "node ./scripts/publish.js",
-    "test": "node packages/eslint-plugin-ckeditor5-rules/tests/index.js && echo OK!"
+    "test": "jest && node packages/eslint-plugin-ckeditor5-rules/tests/index.js"
```

Comment:
I guess we need a dedicated "test" script for the main package that will execute `npm test` in all packages as the repository contains more packages with tests.